lib/inets/test/httpd_SUITE.erl

```diff
@@ -1275,6 +1276,25 @@ content_length(Config) ->
 				       [{statuscode, 200},
 					{content_length, 274},
 					{version, Version}]).
+
+%-------------------------------------------------------------------------
+filename_too_long() ->
+    ["Tests what happens if supplied filename exceeds os-limit of filename characters."].
```

Comment:
While working on #7904 I found there were some different ways to add these doc properties to test cases, with `[{doc, "Test case description"}]` being the most common. Is there some intentional format that the testcase configuration functions try to follow or is it whatever fits best to document it?

Reply:
It should be `{doc, "Some doc"} ` , the the string only is legacy, I happened to copy a legacy case that was not fixed when creating the new one.  I will fix this suites test cases thanks for pointing it out!